scalafmt-tests/src/test/resources/newlines/alwaysBeforeMultilineDefF.stat

```diff
@@ -0,0 +1,49 @@
+maxColumn = 40
```

Comment:
Is the rename of this file intentional?

Reply:
it was, although i can't find the change which contains the test with `T`...

Reply:
What happens if you rename the file back to the original name?

Reply:
done